test/e2e/framework/test_context.go

```diff
@@ -41,9 +41,9 @@ func registerFlags(t *TestContextType) {
 	flag.StringVar(&TestContext.ReportPrefix, "report-prefix", "", "Optional prefix for JUnit XML reports. Default is empty, which doesn't prepend anything to the default name.")
 	flag.StringVar(&TestContext.ReportDir, "report-dir", "", "Path to the directory where the JUnit XML reports should be saved. Default is empty, which doesn't generate these reports.")
 	flag.StringVar(&TestContext.SubmarinerNamespace, "submariner-namespace", "submariner", "Namespace in which the submariner components are deployed.")
-	flag.UintVar(&TestContext.ConnectionTimeout, "connection-timeout", 6, "The timeout in seconds per connection attempt when verifying communication between clusters.")
+	flag.UintVar(&TestContext.ConnectionTimeout, "connection-timeout", 90, "The timeout in seconds per connection attempt when verifying communication between clusters.")
```

Comment:
This seems rather large - with 10 attempts the total timeout will be 900 sec (like 15 min).  Did you intend for 90 sec total? If so then this would be 9.

Reply:
Then may be we need to reduce the connection attempts. One issue I found is that you need to do this across clusters, and while you are spawning the connector pod, the listener pod could timeout and go away. 

I will reduce the connection attempts.

Reply:
I'm not sure if it's better to have lower timeout and more attempts or vice versa.